lib/multiselect.dart

```diff
@@ -89,7 +92,7 @@ class DropDownMultiSelect<T> extends StatefulWidget {
   final Widget Function(T option)? menuItembuilder;
 
   /// a function to validate
-  final String Function(T? selectedOptions)? validator;
```

Comment:
Here, the rest is auto format and libs update

Reply:
This should be changed to `final String? Function(T? selectedOptions)? validator;` to return null when there is no issue.